examples/test_robotdriver2.py

```diff
@@ -0,0 +1,228 @@
+import unittest
+
+from edg import *
+from .test_robotdriver import LipoConnector, MotorConnector, PwmConnector
+
+
+class LedConnector(Block):
```

Comment:
Might be a good idea to add a comment here about what this is for, eg it's a connector for external WS2812s.

Might also be useful to add a TODO for the LEDs to be parameterized, and to share parameters with the WS2812 block.